src/test/java/com/tagtraum/perf/gcviewer/imp/TestDataReaderGo.java

```diff
@@ -0,0 +1,39 @@
+package com.tagtraum.perf.gcviewer.imp;
+
+import static org.junit.Assert.assertEquals;
+
+import com.tagtraum.perf.gcviewer.UnittestHelper;
+import com.tagtraum.perf.gcviewer.model.GCModel;
+import com.tagtraum.perf.gcviewer.model.GcResourceFile;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.junit.Test;
+
+public class TestDataReaderGo {
+
+    @Test
+    public void test() throws IOException {
+        String gcLog = ""
+                + "gc starting...\n" // Such a line is not produced by the Go GC; it is just for testing
+                + "gc 1 @997.597s 3%: 68+0.36+0.51 ms clock, 205+0/16/89+1.5 ms cpu, 84->84->42 MB, 86 MB goal, 3 P\n"
```

Comment:
So, the pause for this event is 68 + 0.51 ms. What about the 0.36 ms?

Reply:
The 0.36 ms is part of the GC time. I did not count it since I was only interested in the stop-the-world time. You can add it if you want. As I said above, the GC fields were not documented enough for me to decide which duration to include and which to omit.